### DISCUSSION TITLE
You cannot use mobile payment to buy GitHub copilot, such as Alipay or wechat

### DISCUSSION THREAD

**Discussion by qq987408706** [community/community#31456] [Copilot Conversations]
Payment is particularly inconvenient

**Comment by candyho** [comment]
Hey, thanks for the feedback. Looks like you would like to purchase GitHub Copilot using Alipay or WeChat? I'll transfer this feedback to the Copilot category. The Mobile category is mainly for feedback related to the GitHub Mobile apps.

**Reply by qq987408706** [reply]
thanks

**Reply by CherryLover** [reply]
Hope to join Alipay or WeChat Pay as soon as possible

**Comment by iamcjwilliams** [comment]
@qq987408706 Please visit here [About Billing On GitHub](https://docs.github.com/en/billing/managing-your-github-billing-settings/about-billing-on-github)
